### PR TITLE
UBB - Private Offers for Dedicated

### PR DESCRIPTION
**Pull request by micheleRP**
fixes https://github.com/redpanda-data/documentation-private/issues/2196 

This covers private offers for Dedicated clusters on GCP and AWS. 

preview: 
- [Manage Billing index page](https://deploy-preview-251--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/manage-billing/)
- [Consumption Metrics](https://deploy-preview-251--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/manage-billing/billing/)
- [Use GCP Commits](https://deploy-preview-251--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/manage-billing/gcp-commit/)
- [Use AWS Commits](https://deploy-preview-251--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/manage-billing/aws-commit/)

### CODE REVIEW

File: modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -68,6 +68,12 @@
 *** xref:deploy:deployment-option/cloud/cloud-overview.adoc[Redpanda Cloud Overview]
 *** xref:deploy:deployment-option/cloud/whats-new-cloud.adoc[]
 *** xref:get-started:quick-start-cloud.adoc[Quickstart]
+*** xref:deploy:deployment-option/cloud/manage-billing/index.adoc[Manage Billing]
+**** xref:deploy:deployment-option/cloud/manage-billing/billing.adoc[Marketplace Consumption Metrics]
+**** xref:deploy:deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc[]
```
Comment (greg-szrama) [inline modules/ROOT/nav.adoc]:
```suggestion
**** xref:deploy:deployment-option/cloud/manage-billing/gcp-pay-as-you-go.adoc[Use with GCP Pay As You Go]
```

File: modules/deploy/pages/deployment-option/cloud/manage-billing/aws-commit.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
+= Use with AWS Commitment
+:description: Subscribe to Redpanda Cloud in AWS Marketplace with discounted commitment billing.
+
+With a usage-based billing commitment, you sign up for a minimum spend amount and get a discount on your committed usage. This allows you to subscribe to Redpanda Cloud through AWS Marketplace to quickly provision Redpanda Dedicated clusters. 
+
+Commitments are minimums. If you use more than your committed amount, you can continue using Redpanda Cloud without interruption. You will be charged at your discounted rate for usage beyond the committed amount until the end of your commitment term. 
+
+[IMPORTANT]
+====
+When you sign up for Redpanda Cloud through AWS Marketplace, you can only create clusters on AWS.
```
Comment (greg-szrama) [inline modules/deploy/pages/deployment-option/cloud/manage-billing/aws-commit.adoc]: I'm reading this as, "You can only use AWS Marketplace to create AWS Clusters." A user could still use GCP to set up a GCP cluster, right? Or will Redpanda licensing not support that?

Mainly just looking for some clarification on this!

File: modules/deploy/pages/deployment-option/cloud/manage-billing/index.adoc [hunk 3]
```diff
@@ -0,0 +1,4 @@
+= Manage Billing
+:description: Learn about Redpanda consumption metrics and subscriptions to Redpanda Cloud with discounted commitment billing.
```
Comment (towfiqa) [inline modules/deploy/pages/deployment-option/cloud/manage-billing/index.adoc]: Learn about how Redpanda measures consumption through usage metrics in Redpanda Cloud.

File: modules/deploy/pages/deployment-option/cloud/manage-billing/aws-commit.adoc (hunk 2 above)
Comment (towfiqa) [reply modules/deploy/pages/deployment-option/cloud/manage-billing/aws-commit.adoc]: Users will only be able to create clusters in AWS through AWS Marketplace. Similary they will be able to create clusters in GCP through GCP Marketplace only.  They won't be able to create an AWS cluster through GCP Marketplace for example.

File: modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc [hunk 4]
```diff
@@ -0,0 +1,29 @@
+= Consumption Metrics
+:description: Learn about the consumption units in Redpanda Cloud billing.
+
+Redpanda Cloud billing is mapped to your consumption of resources. When you create a cluster, you select a xref:deploy:deployment-option/cloud/cloud-overview.adoc#cluster-tiers[throughput tier] for expected data ingress and egress, partitions, and connections. Billing is based on the number of units consumed is multiplied by the cost for that unit.
```
Comment (towfiqa) [inline modules/deploy/pages/deployment-option/cloud/manage-billing/billing.adoc]: Replace throughput tier] for expected data ingress and egress, partitions, and connections. Billing is based on the number of units consumed is multiplied by the cost for that unit

WITH
Cluster Tier which specifies the expected throughput for you cluster including ingress, egress, number of partitions, and connections limits. Billing is based on the number of units consumed, multiplied by the cost for that unit.

File: modules/deploy/pages/deployment-option/cloud/manage-billing/aws-commit.adoc [hunk 5]
```diff
@@ -0,0 +1,43 @@
+= Use AWS Commits
+:description: Subscribe to Redpanda in AWS Marketplace with discounted commitment billing.
+
+With a usage-based billing commitment, you sign up for a minimum spend amount and get a discount on your committed usage. This allows you to subscribe to Redpanda Cloud through AWS Marketplace to quickly provision Redpanda Dedicated clusters. 
+
+Commitments are minimums. If you use more than your committed amount, you can continue using Redpanda Cloud without interruption. You will be charged at your discounted rate for usage beyond the committed amount until the end of your commitment term. 
+
```
Comment (towfiqa) [inline modules/deploy/pages/deployment-option/cloud/manage-billing/aws-commit.adoc:10]: Let's also add:

If your usage is below your commitment minimum, you are still expected to pay the minimum but a rollover occurs for the unused amount which is set to expire for the end of contract.

(Feel free to wordsmith this some)

File: modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-commit.adoc [hunk 6]
```diff
@@ -0,0 +1,43 @@
+= Use GCP Commits
+:description: Subscribe to Redpanda in Google Cloud Marketplace with discounted commitment billing.
+
+With a usage-based billing commitment, you sign up for a minimum spend amount and get a discount on your committed usage. This allows you to subscribe to Redpanda Cloud through Google Cloud Marketplace to quickly provision Redpanda Dedicated clusters. 
+
+Commitments are minimums. If you use more than your committed amount, you can continue using Redpanda Cloud without interruption. You will be charged at your discounted rate for usage beyond the committed amount until the end of your commitment term. 
+
```
Comment (towfiqa) [inline modules/deploy/pages/deployment-option/cloud/manage-billing/gcp-commit.adoc:10]: ADD:

If your usage is below your commitment minimum, you are still expected to pay the minimum but a rollover occurs for the unused amount which is set to expire for the end of contract.

(Feel free to wordsmith this some)

File: modules/deploy/pages/deployment-option/cloud/manage-billing/index.adoc (hunk 3 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/manage-billing/index.adoc]: @towfiqa, how about: Learn about the metrics Redpanda uses to measure consumption in Redpanda Cloud.

File: modules/deploy/pages/deployment-option/cloud/manage-billing/aws-commit.adoc (hunk 5 above)
Comment (micheleRP) [reply modules/deploy/pages/deployment-option/cloud/manage-billing/aws-commit.adoc:10]: @towfiqa, please [see revision](https://deploy-preview-251--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/cloud/manage-billing/gcp-commit/) (for both AWS and GCP).

**Review by towfiqa — APPROVED** [review]

**Review by gavinheavyside — APPROVED** [review]

**Review by JakeSCahill — APPROVED** [review]
LGTM

**Pull request merged**